test/node/schemas/vis-state-schema-test.js

```diff
@@ -131,8 +131,50 @@ test('#visStateSchema -> v1 -> save load interaction', t => {
     tooltip: {
       enabled: true,
       fieldsToShow: {
-        [testCsvDataId]: ['gps_data.utc_timestamp', 'gps_data.types', 'epoch', 'has_result', 'id'],
-        [testGeoJsonDataId]: ['OBJECTID', 'ZIP_CODE', 'ID', 'TRIPS', 'RATE']
+        [testCsvDataId]: [
```

Comment:
Can you add a test case saving an interaction with specific `format` and load it back?

Reply:
Yes